test/patterns/return_empty_string/test_return_empty_string.py

```
@@ -0,0 +1,37 @@
+from aibolit.utils.ast_builder import build_ast
+
+
+class ReturnEmptyStringPatternTestCase(TestCase):
+    current_directory = Path(__file__).absolute().parent
+
+    def test_simple(self):
+        filepath = self.current_directory / 'Simple.java'
+        ast = AST.build_from_javalang(build_ast(filepath))
+        pattern = ReturnEmptyString()
+        lines = pattern.value(ast)
+        # В файле Simple.java return "" на 6-й строке
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF003)